R/init.R

```diff
@@ -89,6 +90,13 @@ build_site_meta <- function(pkg = ".") {
   )
 
   if (!is.null(pkg$meta$url)) {
+    # Add a CNAME file if it doesn't exist
```

Comment:
Could create if not present be an argument to `build_cname()` to reduce the amount of duplicated code a little

Reply:
Oh actually I can remove this code completely, it was from my first attempt before I decided to write build_cname